pkg/otlp/metrics/metrics_remapping_test.go

```diff
@@ -275,16 +322,20 @@ func TestRemapMetrics(t *testing.T) {
 			out: []pmetric.Metric{metric("container.net.rcvd.packets", point{f: 15})},
 		},
 	} {
-		lena := dest.Len()
-		checkprefix := strings.HasPrefix(tt.in.Name(), "system.") || strings.HasPrefix(tt.in.Name(), "process.")
-		remapMetrics(dest, tt.in)
-		if checkprefix {
-			require.True(t, strings.HasPrefix(tt.in.Name(), "otel."), "system.* and process.* metrics need to be prepended with the otel.* namespace")
-		}
-		require.Equal(t, dest.Len()-lena, len(tt.out), "unexpected number of metrics added")
-		for i, out := range tt.out {
-			require.Equal(t, out, dest.At(dest.Len()-len(tt.out)+i))
-		}
+		t.Run("", func(t *testing.T) {
+			dest := pmetric.NewMetricSlice()
```

Comment:
It would be more clear if `in` is in `dest` which is consistent with:
https://github.com/DataDog/opentelemetry-mapping-go/blob/3d07b09ccf1829bd6bbe804f65d5ee6756478776/pkg/otlp/metrics/metrics_translator.go#L661

https://github.com/DataDog/opentelemetry-mapping-go/blob/3d07b09ccf1829bd6bbe804f65d5ee6756478776/pkg/otlp/metrics/metrics_translator.go#L682